examples/feerate.rs

```diff
@@ -0,0 +1,87 @@
+//! # Fee Rate Monitoring Example
+//!
+//! This example demonstrates how to use the bitcoin indexer to monitor estimated fee rates
+//! from a Bitcoin node. It's designed to work with an external Bitcoin node configured
+//! through the `config/development.yaml` file.
+//!
+//! ## Configuration
+//!
+//! The example reads configuration from `config/development.yaml` and can be used with:
+//! regtest, testnet, or mainnet (bitcoin).
+//!
+//! When using regtest, ensure your regtest blocks are been filled with at least 6 transactions
+//!
+//! ## Usage
+//!
+//! 1. Configure your Bitcoin node connection in `config/development.yaml`
+//! 2. Ensure your Bitcoin node is running and accessible
+//! 3. Run the example: `cargo run --example feerate`
+//! 4. Observe the fee rate estimates printed to the console
+//! 5. Press Ctrl+C to gracefully shutdown
+//!
+//! ## Behavior
+//!
+//! The example will:
+//! - Connect to the configured Bitcoin node
+//! - Start the indexer and continuously sync with the blockchain
+//! - Query and display estimated fee rates every 50ms
+//! - Handle graceful shutdown on Ctrl+C
+//!
+
+use anyhow::Result;
+use bitcoin_indexer::{
+    config::IndexerConfig,
+    indexer::{Indexer, IndexerApi},
+    store::IndexerStore,
+};
+
+use bitvmx_bitcoin_rpc::{
+    bitcoin_client::{BitcoinClient, BitcoinClientApi},
+    types::BlockHeight,
+};
+use bitvmx_settings::settings;
+use std::{rc::Rc, thread::sleep};
+use storage_backend::storage::Storage;
+use tracing::info;
+
+fn main() -> Result<(), anyhow::Error> {
```

Comment:
do not use main, use test instead.

Reply:
it is using main cause it is inside an example, which i use to run standalone and compare long runs again mempool.space estimator in mainnet, so i stick with main